ERCS/erc-7714.md

```diff
@@ -0,0 +1,68 @@
+---
+eip: 7714
+title: Simple Permissions Checks
+description: Standardized interface for checking user permissions
```

Comment:
Your description could be a bit more detailed. What is a permission? What makes this proposal "simple"?